gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/login.html

```diff
@@ -54,16 +57,19 @@
                            th:name="${bot_detection_configuration.get('tokenParameterName')}" />
 
                     <div class="login-form-actions">
+                        <div th:if="${identifierFirstLoginEnabled}" class="mdl-cell mdl-cell--10-col mdl-typography--text-left no-margin-left">
+                            <a href="javascript:history.back()">Back to login</a>
```

Comment:
Not sure about the Back to login title ..., maybe Switch account ? 

And I think we add this link with the other ones (Forgot password, etc ...) sorry for that :D